internal/client/client.go

```diff
@@ -14,12 +14,34 @@ import (
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/clientcredentials"
 
+	"encoding/base64"
+	"encoding/json"
+
 	"github.com/auth0/go-auth0"
 )
 
 // UserAgent is the default user agent string.
 var UserAgent = fmt.Sprintf("Go-Auth0-SDK/%s", auth0.Version)
 
+// Telemetry is the structure used to send telemetry data in the "Auth0-Client" header.
+type Telemetry struct {
+	Name    string            `json:"name"`
+	Version string            `json:"version"`
+	Env     map[string]string `json:"env,omitempty"`
+}
+
+// IsEmpty checks whether the provide Telemetry data is nil or has no data to allow
```

Comment:
```suggestion
// IsEmpty checks whether the provided Telemetry data is nil or has no data to allow
```

Reply:
Resolved in c336a83a92ef478bbe2e7de232ed68a0751f5431